website/server/models/group.js

```diff
@@ -516,13 +516,17 @@ schema.methods.isMember = function isGroupMember (user) {
   return user.guilds.indexOf(this._id) !== -1;
 };
 
-schema.methods.getMemberCount = async function getMemberCount () {
+schema.methods.getMemberCount = async function getMemberCount (excludeUserId) {
```

Comment:
This might be more intuitive as an options object with an `excludeUserId` property -- I was confused at first seeing this parameter elsewhere and being like "why are we sending a user ID to a group member count function?" hah

Reply:
good Idea! I did that.